templates/dashboard/order/detail.html

```diff
@@ -304,6 +307,23 @@ <h5 class="data-table-title">
                       <td>
                         {{ item.product_name }}<br>
                       </td>
+                      <td class="right-align">
+                        {% if group.status != 'cancelled' %}
```

Comment:
The status value is hardcoded here. Instead, there should be a method in the model that will check this condition (in more complex cases in templatetag). Here something like `can_change_stock` in the model should be fine.